src/main/scala/cromwell/binding/FinalCall.scala

```diff
@@ -0,0 +1,38 @@
+package cromwell.binding
```

Comment:
Did you consider putting this in `cromwell.engine` instead?  It feels out of place a little bit because `FinalCall`s can't be created from WDL source